test/parallel/test-fs-watch-close-when-destroyed.js

```diff
@@ -4,6 +4,10 @@
 // already destroyed will result in a noop instead of a crash.
 
 const common = require('../common');
+
+if (common.isIBMi)
+  common.skip('IBM i does not support `fs.watch()` by now');
```

Comment:
```suggestion
  common.skip('IBM i does not support `fs.watch()`');
```

Can you drop "by now" throughout the PR.